test/ModalSpec.js

```diff
@@ -1,18 +1,31 @@
 import React from 'react';
 import ReactTestUtils from 'react/lib/ReactTestUtils';
 import Modal from '../src/Modal';
-import { shouldWarn } from './helpers';
+import { render } from './helpers';
 
 describe('Modal', function () {
+  let mountPoint;
+
+  beforeEach(()=>{
+    mountPoint = document.createElement('div');
+    document.body.appendChild(mountPoint);
+  });
+
+  afterEach(function () {
+    React.unmountComponentAtNode(mountPoint);
+    document.body.removeChild(mountPoint);
+  });
 
   it('Should render the modal content', function() {
     let noOp = function () {};
-    let instance = ReactTestUtils.renderIntoDocument(
-      <Modal onHide={noOp}>
+    let instance = render(
+      <Modal show onHide={noOp} animation={false}>
```

Comment:
heh... I missed `<Modal />` addition..
I would propose `disableAnimation` instead of `animation`,
then use could just write it as:
```js
<Modal show onHide={noOp} disableAnimation>
```

though, this isn't really significant